.github/workflows/lintern.yml

```diff
@@ -0,0 +1,13 @@
+name: Lintern
+on: [push]

```

Comment:
@alberturria do you mind updating so we can also run the CI on `pull_request`?

Reply:
Sure, on it! 🏃